docs/overview.rst

```diff
@@ -102,3 +102,28 @@ For example, the `Escrow` contract factory requires linking to an instance of th
        deployer.register_strategy("Escrow", escrow_strategy)
        linked_escrow_package, _ = deployer.deploy("Escrow") 
        return linked_escrow_package.deployments.get_deployment("Escrow")
+
+
+Log
+---
+
+The ``Log`` class is available to help with testing for contract events, and the contents of the emitted logs.
+
+
+``tests/fixtures/ping.vy``
+
+.. include:: ../tests/fixtures/ping.vy
+   :code: python
+
+
+.. code:: python
+
+   # SETUP
+   ping_deployer = vy_deployer.deploy("ping")
+   ping_instance = ping_deployer.deployments.get_contract_instance("ping")
+   tx_hash = ping.functions.ping(b"one", b"two")
```

Comment:
I think this should be `ping_instance.functions.ping(b"one", b"two").transact()`

Reply:
i've been trying to figure out a way to doctest the deployer fixtures, which would ideally eliminate these errors, had some trouble figuring it out, but it's on the todo list and i hope to get around to it soon

Reply:
http://www.sphinx-doc.org/en/stable/usage/extensions/doctest.html#directive-testsetup

This might be what you're looking for.  It allows you to write some code that doesn't render in the docs but which is part of the execution scope of the code block.  Not sure if that solves your problem...